### PR TITLE
Update ShippingForm.js

### PR DESCRIPTION
**Pull request by tacoelho**
Show Shipping method validation error to customer #114

### CODE REVIEW

**Comment by jaepass** [comment]
thanks will get to testing this out!

**Comment by tacoelho** [comment]
> 
> 
> thanks will get to testing this out!

yay!

File: components/checkout/common/ShippingForm.js [hunk 1]
```diff
@@ -188,6 +189,8 @@ export default class ShippingForm extends Component {
               </p>
               <Dropdown
                 name="fulfillment[shipping_method]"
+                required={true}
+                validationMessage={shippingValidationMessage}
```
Comment (jaepass) [inline components/checkout/common/ShippingForm.js:193]: this prop is not being defined anywhere currently

**Review by jaepass — COMMENTED** [review]
Also might be good to work it in the checkout page errors somehow that we have here https://github.com/chec/commercejs-nextjs-demo-store/blob/9477fba13bdc03aab38ea90287b7c07d531ea737/pages/checkout/index.js#L23

**Comment by tacoelho** [comment]
> Also might be good to work it in the checkout page errors somehow that we have here
> 
> https://github.com/chec/commercejs-nextjs-demo-store/blob/9477fba13bdc03aab38ea90287b7c07d531ea737/pages/checkout/index.js#L23

Ok, I will.

**Comment by jaepass** [comment]
Hi @tacoelho, this is not working as intended and does not show any validation message as outlined in the original issue. I will close this for now but if you're still keen on exploring other options feel free to open another pr. We're happy to also accept the enabling of a disabled class if the shipping option is not selected, and active when is it selected

**Pull request closed without merging**